crates/web-sys/webidls/unstable/LocalFontAccess.webidl

```diff
@@ -0,0 +1,20 @@
+[SecureContext]
+partial interface Window {
+  [Throws]
+  Promise<sequence<FontData>> queryLocalFonts(optional QueryOptions options = {});
```

Comment:
(just self-documenting)
Throwing behavior is documented in the specification.
Implementation: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/font_access/window_font_access.idl.